src/Organisms/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import CardWrapper from '../../Atoms/CardWrapper/CardWrapper';
+import RegistrationFormStyled from './RegistrationFormStyled';
+import InputField from '../../Molecules/InputField/InputField';
+import CheckboxAgreeField from '../../Atoms/CheckboxAgreeField/CheckboxAgreeField';
+
+const RegistrationPage = () => {
+  const [name, setName] = useState('');
+  const [surname, setSurname] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [street, setStreet] = useState('');
+  const [zipcode, setZipcode] = useState('');
+  const [city, setCity] = useState('');
+  const navigate = useNavigate();
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+
+    const data = { name, surname, email, password, street, zipcode, city };
+
+    //test, after submit, entered user data
+    console.log(data);
+    navigate('/login');
+  };
+
+  return (
+    <CardWrapper>
+      <RegistrationFormStyled onSubmit={submitHandler}>
+        <InputField
+          labelText="Email"
+          htmlFor="email"
+          type="email"
+          placeholder="&#xf0e0; Email"
+          id="email"
+          required
+          value={email}
+          onChange={(event) => setEmail(event.target.value)}
+        />
+
+        <InputField
+          labelText="Password"
```

Comment:
Also remmber about repeated password and to check if they match. If there is any rules you want to implement for passwords, this would be a good place to start validating process